src/main/resources/templates/smallfull/creditorsAfterOneYear.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+
+<html
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="http://www.w3.org/1999/xhtml"
+    layout:decorate="~{layouts/baseLayout}">
+
+<head>
+  <title>Creditors - amounts falling due after one year note</title>
+</head>
+
+<div id="creditors-after-one-year-main-content" layout:fragment="content" class="two-column-accounts">
+
+  <form id="balanceSheet-form" th:action="@{''}" th:object="${creditorsAfterOneYear}" class="form currency-included-within-inputs govuk-body" method="post">
+
+    <div th:replace="fragments/globalErrors :: globalErrors"></div>
+
+    <div th:replace="fragments/numberedHeading :: numberedHeading (
+                    headingText = 'Creditors - amounts falling due after one year note'
+                )">
+    </div>
+
+    <div class="left-indent">
+      <div class="panel panel-border-wide">
+        <p id="creditors-after-one-year-help-text" class="warning-note warning-note-large">You only need to enter items included in your prepared accounts.</p>
+      </div>
+
+      <div class="govuk-grid-row" id="accounts-header">
+        <div class="govuk-grid-column-one-half">
+          &nbsp;
+        </div>
+        <div class="govuk-grid-column-one-quarter">
+          <h2 class="govuk-heading-s">
+            <span id="currentPeriodHeading" th:text='*{balanceSheetHeadings.currentPeriodHeading}'>Current Period</span>
+            <!-- Hidden input for form binding -->
+            <input th:field="*{balanceSheetHeadings.currentPeriodHeading}" type="hidden">
+          </h2>
+        </div>
+        <div class="govuk-grid-column-one-quarter">
+          <h2 class="govuk-heading-s">
+            <span id="previousPeriodHeading" th:text='*{balanceSheetHeadings.previousPeriodHeading}'>Previous Period</span>
```

Comment:
Just wondering, what would happen if it is a first year filer?
`balanceSheetHeadings.previousPeriodHeading` will be null, wont it? and therefore, previous year column should not be displayed. 

Would previous year be displayed?